### ISSUE TITLE
Number of samples option

### ISSUE THREAD

**Issue by zanini007** [pushrax/OpenVR-SpaceCalibrator#71]
By default, we're allowed to use the fast, slow and very slow calibrations that uses 100, 250 and 500 samples...
But sometimes i feel like the slow (250) are not enough (imprecise) and veryslow (500) might be too much and i'm wasting time.

Also, by messing around in visual studio, i've even tried to use 750 samples or above just to test and it made easier to calibrate with nom-ideal environments (but not ideal and takes too much time, better just redo the calibration but the right way than implementing that)

So basically my sugestion is giving something like an advanced option to literraly choose the number of samples you want to use for the next calibration, similar to the ones in the edit calibration screen.
